lib/tasks/cuttlefish.rake

```diff
@@ -16,6 +16,18 @@ namespace :cuttlefish do
     Email.all.each {|email| email.update_status! }
   end
 
+  desc "Archive all emails created more than 6 months ago"
+  task :archive_6_month_old_emails => :environment do
+    date_of_oldest_email = Delivery.order(:created_at).first.created_at.to_date
+    if date_of_oldest_email < 6.months.ago.to_date
```

Comment:
`6.month.ago` is repeated and you can DRY this up. Reducing the duplication will make it clearer, less error-prone, and easier to modify in the future.